pkg/platform/model/vcs.go

```diff
@@ -139,6 +140,10 @@ func (t NamespaceType) Prefix() string {
 	return t.prefix
 }
 
+func (t NamespaceType) Matchable() NamespaceMatchable {
+	return t.matchable
+}
```

Comment:
This mechanic feels redundant to me. Can't we just match the actual types? ie. `nsTypeIHave == NamespacePackage`.

Reply:
For the `FilterCheckpointNamespace` function the namespace we get back is a string. `NamespaceMatch()` and the `NamespaceMatchable` type make that comparison fairly straightforward.